libsolidity/interface/CompilerStack.cpp

```diff
@@ -1286,7 +1286,7 @@ void CompilerStack::assemble(
 			"Contract code size is "s +
 			to_string(compiledContract.runtimeObject.bytecode.size()) +
 			" bytes and exceeds 24576 bytes (a limit introduced in Spurious Dragon). "
-			"This contract may not be deployable on Mainnet. "
```

Comment:
Bad rebase?

Reply:
Yeah, maybe, not sure what happened here ...